twins/api/twins/http/doc.go

```diff
@@ -0,0 +1,9 @@
+//
```

Comment:
There is no need for `twins/api/twins/http/`, `twins/api` is sufficient.

Reply:
Moved to `twins/api/http`.